### PR TITLE
README: Added mention about registering command class in dic (config.neon)

### PR DESCRIPTION
**Pull request by paveljanda**

### CODE REVIEW

File: .docs/README.md [hunk 1]
```diff
@@ -70,7 +70,12 @@ console:
 
 ## Command
 
+### First, you have to define the command class:
```
Comment (f3l1x) [inline .docs/README.md]: `### Create command`

File: .docs/README.md [hunk 2]
```diff
@@ -101,6 +106,15 @@ final class FooCommand extends AbstractCommand
 }
 ```
 
+### Second, the command needs to be registered in the `dic` (`config.neon`):
```
Comment (f3l1x) [inline .docs/README.md]: `### Register command`

**Comment by paveljanda** [comment]
Sure, done

**Comment by f3l1x** [comment]
Best. 👍

**Pull request merged**